package-lock.json

```diff
@@ -9,9 +9,9 @@
       "integrity": "sha1-sC0QqwKOKSisWSoFGqpJgaGUHQM="
     },
     "@types/node": {
-      "version": "6.0.78",
```

Comment:
I haven't messed with this one at all. I think it's created automatically, so I'll not touch it.